### DISCUSSION TITLE
sono bloccato sulla schermata "preparazione dell'assistente domestico"

### DISCUSSION THREAD

**Discussion by Gianfycc** [indomus/forum#2140] [HUB: Home Assistant]
#salve. premetto che ho persino formattato il pc per evitare che non ci fossero errori. 
ho installato visual c++ 19 come consigliato.
detto ciò da ieri ho eseguito mille prove di installazione di home assistant os con virtualbox.
sono certo di aver fatto tutto bene perchè tutte i video da me visti nonchè le guide lette sono tutte le stesse. 
Effettivamente una volta fatto invio sulla MV sembra tutto ok. Solo che poi procedo mettendo su browser (provato si chroome che microsoft edge) a mettere indirizzo ip:8123, parte la pagina con su scritto PREPARAZIONE DELL'ASSISTENTE DOMESTICO (questo può richiedere fino a 20 minuti; in realta non si schioda da li. 
Ho provato sia sulla rete fissa di casa con router windtre che rifacendo il tutto e usando lo smartphone come hotspot.
Cosa posso fare? sono veramente demoralizzato.

**Comment by 50494554524F** [comment]
mettiti comodo ed aspetta che finisca, ci vuole parecchio tempo perché deve scaricare gli aggiornamenti...

**Comment by Gianfycc** [comment]
È acceso che procede. Se è così lo lascio acceso tutta notte perché Internet sta facendo schifo in questo due giorni. spero bene. Grazie
Domani ti aggiorno
